### PR TITLE
Hard rate limit submissions

### PR DESCRIPTION
**Pull request by dannyroberts**
##### SUMMARY
Adds a toggle we can use to rate limit specific domains, as well as to turn it on for all domains if we want (and off again quickly in response to any issues).

See also https://github.com/dimagi/commcare-hq/pull/25982 (analogous change but for restores)

##### FEATURE FLAG
RATE_LIMIT_SUBMISSIONS_AND_RESTORES

##### PRODUCT DESCRIPTION
Allows the internal tech team / support to turn off and on hard rate-limiting of submissions dynamically.

### CODE REVIEW

File: corehq/apps/toggle_ui/static/toggle_ui/js/flags.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ hqDefine('toggle_ui/js/flags', [
             if (viewModel.tagFilter() === 'all') {
                 return true;
             }
-            var tag = aData[0].replace(/\n/g," ").replace(/<.*?>/g, "");
+            var tag = aData[0].replace(/\s+/g," ").replace(/<.*?>/g, "").replace(/^\d+ /, "");
```
Comment (millerdev) [inline corehq/apps/toggle_ui/static/toggle_ui/js/flags.js]: Is the final replace (`.replace(/^\d+ /, "")`) intended to replace more than the first occurrence/match? If yes it needs a `g` flag on the regexp.

**Review by millerdev — APPROVED** [review]

File: corehq/apps/toggle_ui/static/toggle_ui/js/flags.js (hunk 1 above)
Comment (dannyroberts) [reply corehq/apps/toggle_ui/static/toggle_ui/js/flags.js]: Nope, it's just intended to strip numbers from the beginning of the line. The numbers were introduced recently to fix sort order, which broke filtering.

I wish we did this in a cleaner way, but it wasn't my focus in this PR, just something I came across while testing.

**Comment by orangejenny** [comment]
Thank you for fixing the UI filtering issue! In case you haven't seen it, https://dimagi-dev.atlassian.net/browse/SAASP-10151 is filed about that.

**Comment by dannyroberts** [comment]
Thanks @orangejenny, I hadn't seen that. I'll update there. Bonus bug fix!

File: corehq/apps/toggle_ui/static/toggle_ui/js/flags.js (hunk 1 above)
Comment (millerdev) [reply corehq/apps/toggle_ui/static/toggle_ui/js/flags.js]: ha, I missed the `^` ... which makes it obvious it's only matching the first occurrence. Sorry for the noise.

**Pull request merged**